.github/workflows/ci.yml

```diff
@@ -406,6 +406,8 @@ jobs:
     env:
       MODIN_ENGINE: ${{matrix.engine}}
       MODIN_MEMORY: 1000000000
+      RAY_LOG_LEVEL: DEBUG
```

Comment:
why?

Reply:
We used this to help debug Ray errors in CI, I will revert.